RadixWallet/Features/CardCarousel/CardCarousel+View.swift

```diff
@@ -0,0 +1,194 @@
+import Foundation
+import SwiftUI
+
+// MARK: - CardCarousel.View
+extension CardCarousel {
+	public struct View: SwiftUI.View {
+		public let store: StoreOf<CardCarousel>
+
+		private let margin: CGFloat = .medium1
+		private let spacing: CGFloat = .small1 * 0.5
+		@ScaledMetric private var height: CGFloat = 105
+		@SwiftUI.State private var selectedCardIndex = 0
+
+		public var body: some SwiftUI.View {
+			WithPerceptionTracking {
+				VStack(spacing: 8) {
+					GeometryReader { _ in
```

Comment:
Is this GeometryReader still needed?

Reply:
Without the dummy cards, and hence the geometry reader. the carousel will not look like Aftab's design, it will not have teaser cards on the sides. Did he change his mind about that?

Reply:
Zeplin [still shows](https://app.zeplin.io/project/639c7e7ab391bb15ac5d25fa/screen/666b6f35261692048fbc7929) teaser cards.

Reply:
We persuaded Aftab to remove these, as they were redundant in the context of showing the position indicator.

Reply:
What a shame, I spent at least 75% of the time on getting teaser cards to work properly. Plus I think it looks much better an indicator, that feels pretty stale nowadays.

Reply:
I also liked the teaser approach better, but it was showing a glitch on first load. Also, solution adds a lot of noise to the code to achieve something not natively supported.

Reply:
I never saw any glitch, was that present in my branch? That should be trivial to fix. I can wrap it all up in a TeasedPagerView when I get back.